src/fork_gc.c

```diff
@@ -1334,6 +1334,8 @@ static int periodicCb(void *privdata) {
     gc->retryInterval.tv_sec = RSGlobalConfig.gcConfigParams.forkGc.forkGcRetryInterval;
     IndexSpecRef_Release(early_check);
     RedisModule_ThreadSafeContextUnlock(ctx);
+    close(gc->pipe_read_fd);
```

Comment:
We need to test it. See `set_tight_maxmemory_for_oom` in `test_gc_oom` and can count and compare file descriptors?